src/main/resources/META-INF/plugin.xml

```diff
@@ -94,6 +94,8 @@
   <change-notes><![CDATA[
     <h2>New Features</h2>
       <ul>
+        <li>Restricted users on free plan to only one private repository</li>
```

Comment:
I think @saleem-latif  was adding to the bottom of this list so probably should use same (I am fine either way but it should be same for every update)

Reply:
Yeah let's follow the convention that has been being used originally